### DISCUSSION TITLE
Again question about bot badge

### DISCUSSION THREAD

**Discussion by kilroy98** [night/betterttv#6620] [Q&A] [answered]
I see that moobot has a bot icon by default. But it still doesn’t work, including for moobot.
https://github.com/night/betterttv/blob/ce08c8a88ffef7e902b8688766fce88a91f56f30/src/modules/chat/index.js#L102

I added dotabod to the list of bots and it still doesn't have an icon. I think that something is broken, which is why the icons for bots do not work.

![Screenshot_1](https://github.com/night/betterttv/assets/13346898/f19f38a1-aebf-4557-b0bb-67b9753d9d8e)

There are no errors in the browser console that are associated with the BetterTTV extension.

- OS: Windows 10
- Browser: Google Chrome 121.0.6167.140
- Version: 7.5.15

**Comment by night** [comment, marked as answer]
I am not able to reproduce any issues with bot badges. Did you check to make sure you do not have other browser extensions installed? Other extensions can "take over" the chat, meaning our features may not work properly.

**Reply by kilroy98** [reply]
Of the extensions, only uBlock Origin (disabled for Twitch), Shazam and BetterTTV are active :(

**Reply by kilroy98** [reply]
@night i took this screenshot on ttv/yapzordota and not see bot badge. What am I doing wrong?
![Screenshot_1](https://github.com/night/betterttv/assets/13346898/50b9b870-141c-44c0-83e3-c9a186fe283d)
